content/oci/deletableoci.go

```diff
@@ -0,0 +1,336 @@
+/*
+Copyright The ORAS Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
nit: The license header is inconsistent with the ones in other files. See https://github.com/oras-project/oras-go/blob/main/content/oci/oci.go#L6-L8.

Reply:
thanks for catching. Adjusted the header statement to be concisistent.